account_move_update_analytic/wizards/account_move_update_analytic_view.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2022 Le Filament
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record id="view_account_move_update_analytic_account" model="ir.ui.view">
+        <field name="name">account.move.update.analytic.form</field>
+        <field name="model">account.move.update.analytic.wizard</field>
+        <field name="arch" type="xml">
+            <form string="Update analytic lines">
+                <group
+                    string="Analytic Account"
+                    groups="analytic.group_analytic_accounting"
+                >
+                    <group>
+                        <field name="current_analytic_account_id" readonly="1" />
+                    </group>
+                    <group>
+                        <field name="company_id" invisible="1" />
+                        <field
+                            name="new_analytic_account_id"
+                            domain="['|', ('company_id', '=', False), ('company_id', '=', company_id)]"
```

Comment:
`check_company=True` is present in `new_analytic_account_id` field. It's really needed?

Reply:
I am not sure, I should try without, but this is how it is done on Odoo code here : https://github.com/odoo/odoo/blob/a99055180c6c8c87603d4beca2156c9e6898ffa4/addons/account/views/account_move_views.xml#L756-L764